apps/cyberstorm-storybook/stories/components/Toast.stories.tsx

```diff
@@ -0,0 +1,67 @@
+import { StoryFn, Meta } from "@storybook/react";
+import { Toast, ToastProps } from "@thunderstore/cyberstorm";
+import React from "react";
+import { v4 as uuid } from "uuid";
+
+const meta = {
+  title: "Cyberstorm/Components/Toast",
+  component: Toast,
+} as Meta<typeof Toast>;
+
+const defaultArgs: ToastProps = {
+  id: uuid(),
+  message: "-",
+  variant: "info",
+};
+
+const Template: StoryFn<typeof Toast> = (args) => (
+  <div style={{ width: "500px" }}>
```

Comment:
Just a thought but would it make sense to use the Viewport component here?

Reply:
It is in use, because of Cyberstorm providers being used in preview.js.